notebooks/source/conf.py

```diff
@@ -102,7 +109,7 @@
 html_theme = "sphinx_rtd_theme"
 html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
 # logo
-# html_logo = '_static/img/Pyro_logo_wide.png'
+html_logo = '_static/img/Pyro_logo_wide.png'
```

Comment:
is this case insensitive?
the logo file is named: `pyro_logo_wide.png`

Reply:
hmm..seems like its case insensitive, but let me fix that.